### ISSUE TITLE
Can't drag and drop image with capitalized extension.

### ISSUE THREAD

**Issue by jumpy-lion** [godotengine/godot#61419]
### Godot version

v4.0.alpha8.official [cc3ed63af]

### System information

Linux Mint 20

### Issue description

I used Krita's Export Layers option, which uses capitalized letters in extension name, and couldn't drag and drop images into scene.
Adding Sprite2D and setting texture manually works fine.

### Steps to reproduce

-Import a file with capitalized PNG extension
-Try to drag and drop an image into scene

**Comment by KoBeWi** [comment]
Can't reproduce. Maybe the issue is a corrupted image? Does it work when you rename the file?

**Comment by jumpy-lion** [comment]
> Can't reproduce. Maybe the issue is a corrupted image? Does it work when you rename the file?

Yes, changing the extension to lowercase png makes it possible again to drag and drop the file.

**Comment by iwek7** [comment]
I checked on Linux Mint 20 Ulyana, commit 8e3084cc91. I was able to reproduce issue with new project and default `icon.png` file

**Comment by Nolkaloid** [comment]
Cannot reproduce on b4644e283
